README.md

```diff
@@ -1512,6 +1512,53 @@ would enable the use of a `@fold` on the `adjacent_animal` vertex field of `Foo`
 }
 ```
 
+## Schema Generation
+To generate a GraphQL schema, we first map a database schema into an intermediate representation:
+the `SchemaGraph`.
+
+#### SchemaGraph
+* `SchemaGraph`
+    * Keeps track of a schema's transitive inheritance structure.
+    * Keeps track of a schema's indexes. Indexes are not used in the GraphQL schema generation and 
+      have a provisional API.
+    * Represents a database's schema through `SchemaElement` objects.
```

Comment:
This section reads very much like an API specification, which is fine once the user has decided this is something they need to use, but not helpful when they are in the process of figuring out what this is and whether they need to use it.

For example, `Represents a database's schema through `SchemaElement` objects.` gives no intuition about what aspects of the database schema these `SchemaElement` objects represent. The next sentences start describing the API of `SchemaElement` objects, and the reader is forced to back-solve for their purpose based on the API they provide.

Try to give the reader an overall intuition of what the purpose of a thing is before describing how it works under the hood.